projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -301,7 +301,7 @@ export class TuiTiptapEditorService extends TuiEditor {
         }
     }
 
-    private isEmptyParagraph(json: Object[]): boolean {
+    private isEmptyParagraph(json: JSONContent[] | undefined): boolean {
```

Comment:
Let's make `json` optional and move this to a utility function since it's pure

Reply:
fixed